apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-start-node/ba-decision-graph-start-node.html

```diff
@@ -0,0 +1,17 @@
+<div class="dt-uxd-startnode-grid">
+  <div *ngFor="let startnode of decisionGraphStartNodes, let i = index">
+    <div
+      class="dt-uxd-startnode"
+      [ngClass]="{
+        'dt-uxd-startnode-visited': isSelectedStartnode(i)
+      }"
+      id="{{i}}"
+    >
+      <div
+        class="dt-uxd-startnode-content"
+        [innerHTML]="getSanitizedNodeText(startnode.text)"
+        (click)="selectStartNode(startnode)"
+      ></div>
```

Comment:
It is an interactive element which should be represented by something that is interactive. Using a button here will give you a lot of things for free from the browser (keyboard activation, focusable).#

```suggestion
      <button
        class="dt-uxd-startnode-content"
        [innerHTML]="getSanitizedNodeText(startnode.text)"
        (click)="selectStartNode(startnode)"
      ></button>
```